### ISSUE TITLE
Can't open Dizzyeggg's expansions

### ISSUE THREAD

**Issue by Sneakyfox77** [Gamer2020/PokemonGameEditor#45]
I used your emerald rom base generator to add the three expansions.  I get an error message saying that there are files missing in the obj folder.  Do I just copy the files from the bin to the obj folder?  Would I replace duplicate files or do something else entirely?  I had been trying to add the expansions separately on my own.  I got the battle engine to work fine, but the other 2 give me errors even on a vanilla rom.  I'm using emerald on windows 10.

**Comment by Sneakyfox77** [comment]
Not positive which things I did to get things to work, but I installed the emerald rom base and pge master folders in the same location.  I can now open those expansions from pge if I open the program from the bin folder in the pge master folder.  If I open the program from anywhere else it doesn't work.  The file I open is the test.gba in the battle upgrade folder that's in the bin folder in the emerald rom base master folder.  I tried to add the various features expansion in the build folder where the other expansions are located.  I copied the aforementioned test.gba file and put it in the various features master folder, renamed it BPEE0.gba, updated the build.py file, and then ran the script.  When I try to open the new test.gba file in pge I get selctedindex cannot equal 65535.  My copied and renamed BPEE0gba file gets the same error.  How can I fix this?
